tests/functions/getToken.js

```diff
@@ -22,8 +22,9 @@ module.exports = async () => {
   });
 
   const { data } = response;
-
-  if (data.errors && data.errors[0].message === 'User not found') {
+  if (data.data !== null) {
```

Comment:
Why are modifications to this file included in this PR? Please remove if they are unnecessary.

Reply:
Actually, they are required as some of the tests were not working correctly with the previous piece of code.